lib/cli.js

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env node
 'use strict'
 
+const logger = require('pino')({ level: 'info', prettyPrint: true, base: null, timestamp: false })
 const commist = require('commist')()
 const { needToShowHelp } = require('./man')
 const parseArgs = require('./args');
 
 ['help', 'login', 'whoami', 'publish', 'draft'].forEach(command => {
   const fn = require(`./commands/${command}`)
-  commist.register(command, (args) => {
+  commist.register(command, async (args) => {
     const opts = parseArgs(args)
     if (!needToShowHelp(command, opts)) {
-      fn(opts)
+      try {
+        const out = await fn(opts)
+        if (out) {
+          logger.info(out)
```

Comment:
This is where I'm not convinced: the out object is [this json](https://github.com/fastify/releasify/pull/8/files#diff-e3eecd52b8e957acc4df7269ea191a94R47).

Should I point that we log `out.message`?